ui/deployment-manager-api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/ProcessingTypeDeploymentService.scala

```diff
@@ -0,0 +1,21 @@
+package pl.touk.nussknacker.engine.api.deployment
+
+import pl.touk.nussknacker.engine.api.ProcessVersion
+import pl.touk.nussknacker.engine.graph.EspProcess
+
+import scala.concurrent.Future
+
+trait ProcessingTypeDeploymentService {
+
+  // shouldn't we pass also category as a filter?
+  def getDeployedScenarios: Future[List[DeployedScenarioData]]
+
+}
+
+case class DeployedScenarioData(processVersion: ProcessVersion, deploymentData: DeploymentData, resolvedScenario: EspProcess)
+
+class ProcessingTypeDeploymentServiceStub(deployedScenarios: List[DeployedScenarioData]) extends ProcessingTypeDeploymentService {
```

Comment:
shouldn't it be test code ?

Reply:
We have some stubs / simple implementations  like this in production code. IMO it is quite good pattern, because u don't need to have separate "test-util" module